curvefs/src/metaserver/fsused_manager.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ *  Copyright (c) 2023 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include <unordered_map>
+#include "curvefs/src/metaserver/fsused_manager.h"
+
+namespace curvefs {
+namespace metaserver {
+
+void FsUsedManager::ApplyFsUsedDeltas() {
+    std::vector<FsUsedDelta> fsUsedDeltaDirty;
+    {
+        std::unique_lock<bthread::Mutex> lock_guard(dirtyLock_);
+        fsUsedDeltaDirty = std::move(fsUsedDeltas_);
+        fsUsedDeltas_ = {};
```

Comment:
`fsUsedDeltas_` is empty at this point, so this code is unnecessary, but you can add `assert(fsUsedDeltas_.empty())` to ensure this.

Reply:
fix